### PR TITLE
Logout FR relate #27

### PR DESCRIPTION
**Pull request by amanshawar**

### CODE REVIEW

File: client/src/components/Shared/Title/index.js [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+        .then(() => {
+          history.push('/login-form');
+        });
+    };
+
+
+    render() {
+      let LogoutButton;
+      if (this.state.logged) {
+        LogoutButton = <LogoutB type="button" onClick={this.clicklogout}>logout</LogoutB>;
```
Comment (jema28) [inline client/src/components/Shared/Title/index.js]: Should be 'Log out'.

File: client/src/components/Shared/Title/title.style.js [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
 font-weight: normal;
 
 @media (max-width: 650px) {
   font-size: 30px;
 }
 `;
+
+export const LogoutB = styled.h1`
+font-size: 16px;
+position:absolute;
```
Comment (jema28) [inline client/src/components/Shared/Title/title.style.js]: Indentation habibti!

**Review by jema28 — CHANGES_REQUESTED** [review]

**Review by FidaaHersh — APPROVED** [review]

**Review by Hadeel-Salamin — APPROVED** [review]
great <3

**Review by jema28 — APPROVED** [review]

File: client/src/components/Feedback/index.js [hunk 3]
```diff
@@ -127,7 +127,7 @@ class Feedback extends Component {
 
     return (
       <React.Fragment>
-        <Title />
+        <Title {...this.props} />
```
Comment (orjwan98) [inline client/src/components/Feedback/index.js:130]: Quick question: what is `{...this.props}`

**Review by orjwan98 — APPROVED** [review]

**Pull request merged**